cmd/unused/internal/ui/group_table.go

```diff
@@ -0,0 +1,64 @@
+package ui
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/grafana/unused"
+)
+
+func GroupTable(ctx context.Context, options Options) error {
+	disks, err := listUnusedDisks(ctx, options.Providers)
+	if err != nil {
+		return err
+	}
+
+	if options.Filter.Key != "" {
+		filtered := make(unused.Disks, 0, len(disks))
+		for _, d := range disks {
+			if d.Meta().Matches(options.Filter.Key, options.Filter.Value) {
+				filtered = append(filtered, d)
+			}
+		}
+		disks = filtered
+	}
+
+	if len(disks) == 0 {
+		fmt.Println("No disks found")
+		return nil
+	}
+
+	w := tabwriter.NewWriter(os.Stdout, 8, 4, 2, ' ', 0)
+
+	headers := []string{"PROVIDER", "GROUP_BY", "TYPE", "SIZE_GB"}
```

Comment:
I think `SIZE_GB` is misleading, perhaps I'd rename it to include the word total in it:
```suggestion
	headers := []string{"PROVIDER", "GROUP_BY", "TYPE", "TOTAL_SIZE_GB"}
```

Reply:
I realize now that it's also missing how many disks are in that group, so perhaps it's better something like:
```suggestion
	headers := []string{"PROVIDER", "GROUP_BY", "TYPE", "DISKS_COUNT", "TOTAL_SIZE_GB"}
```

Reply:
sounds good, I will update code accordingly, thank you!